### PR TITLE
chore: UI issue fixes from demo/v2

### PR DESCRIPTION
**Pull request by maneeshaxyz**
## Summary

Brings demo ui changes to main.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made

- Frontend changes.

## Testing

- [X] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [ ] All existing tests pass

## Checklist

- [ ] My code follows the project's style guidelines
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Screenshots/Demo

(If applicable, add screenshots or GIFs to help explain your changes)

### History 
chore: fixes (#489)

chore: demo-v2 UI fixes and cleanup (#494)

* chore: small ui fixes.

* chore: fix ui issues.

* fix: action button fixes

* chore: save state.

* chore: remove total consignment ui.

Demo v2 (#496)

* chore: fixes

* chore: small ui fixes.

* chore: fix ui issues.

* fix: action button fixes

* chore: save state.

* chore: remove total consignment ui.

* ui fixes

* fix issues.

Demo v2 (#497)

* chore: fixes

* chore: small ui fixes.

* chore: fix ui issues.

* fix: action button fixes

* chore: save state.

* chore: remove total consignment ui.

* ui fixes

* fix issues.

* fix demo branch.

add suresh, romesh and gomesh. (#500)

chore: add sort (#501)

## Deployment Notes

(If applicable, include any special deployment instructions or environment considerations)

### CODE REVIEW

File: portals/apps/trader-app/src/components/WorkflowViewer/ActionListView.tsx [hunk 1]
```diff
@@ -32,16 +34,16 @@ export function ActionListView({
   const groups = useMemo(() => {
     return {
       active: filteredSteps.filter((s) => s.state === 'READY' || s.state === 'IN_PROGRESS'),
-      upcoming: filteredSteps.filter((s) => s.state === 'LOCKED'),
-      finished: filteredSteps.filter((s) => s.state === 'COMPLETED' || s.state === 'FAILED'),
+      // upcoming: filteredSteps.filter((s) => s.state === 'LOCKED'),
```
Comment (ginaxu1) [inline portals/apps/trader-app/src/components/WorkflowViewer/ActionListView.tsx]: safe to delete? why comment out?

**Review by ginaxu1 — CHANGES_REQUESTED** [review]

File: portals/apps/trader-app/src/components/WorkflowViewer/ActionListView.tsx (hunk 1 above)
Comment (maneesha-xyz) [reply portals/apps/trader-app/src/components/WorkflowViewer/ActionListView.tsx]: I kept it as a reference while making the changes. It is safe to delete, so I will remove it now.

File: portals/apps/trader-app/src/screens/ConsignmentDetailScreen.tsx [hunk 2]
```diff
@@ -335,45 +330,7 @@ export function ConsignmentDetailScreen() {
...
-              All steps have been completed. Your consignment is ready.
-            </Text>
-          ) : (
-            <Text size="1" color="gray" className="flex items-center gap-1.5">
-              <ClockIcon />
-              Waiting for dependent steps to be completed before you can proceed.
-            </Text>
-          )}
-        </div>
+        {/* Next Steps section removed */}
```
Comment (ginaxu1) [inline portals/apps/trader-app/src/screens/ConsignmentDetailScreen.tsx]: nit: delete such comments in the various Typescript files of `portals/apps/trader-app/src/screens/`

Otherwise looks fine.

**Review by ginaxu1 — CHANGES_REQUESTED** [review]

File: portals/apps/trader-app/src/screens/ConsignmentDetailScreen.tsx (hunk 2 above)
Comment (maneesha-xyz) [reply portals/apps/trader-app/src/screens/ConsignmentDetailScreen.tsx]: fixed

**Review by ginaxu1 — APPROVED** [review]

**Review by sthanikan2000 — APPROVED** [review]
lgtm
